recommender/conf/locale/en/LC_MESSAGES/text.po

```diff
@@ -0,0 +1,64 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
@OmarIthawi this is the generated file after running `make extract_translations`

Reply:
Please remove it and add it to `.gitignore`